owners/src/repo/virtual_repo.js

```diff
@@ -39,7 +39,40 @@ module.exports = class VirtualRepository extends Repository {
    * Fetch the latest versions of all OWNERS files.
    */
   async sync() {
-    await this.findOwnersFiles();
+    let fileChanged = false;
+    const ownersFiles = await this.github.searchFilename('OWNERS');
```

Comment:
Here's an insane idea: when syncing/warming up, download the entire repository as a tar file (https://api.github.com/repos/ampproject/amphtml/tarball) in memory and read the files from there 🙃
(I checked to see if there's a way to list all the files in a repo using the GitHub API but there really doesn't seem to be) // cc @estherkim @rsimha for sanity check on this insane idea

Reply:
That is quite the impressive combination of insane and fully plausible

Reply:
The tar file for the entire repo is ~130 MB in size, so it's not unreasonable to download it during warm-up if you think it's faster than an API call. (You'd have to work around GAE file system restrictions, right?)

Reply:
Doing the tar expansion in-memory and discarding it, once for warm-up, is _probably_ possible in GAE